wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -685,26 +685,21 @@ <h2>Page editor</h2>
         <section id="tabs">
             <h2>Tabs</h2>
 
-            <ul class="tab-nav" data-tab-nav>
-                <li class="active"><a href="#tab1">Tab 1</a></li>
-                <li><a href="#tab2">Tab 2</a></li>
-            </ul>
-
-            <p>Tabs are currently only used following headers, where they often appear merged with the bottom of the header:</p>
-
-            {% include "wagtailadmin/shared/header.html" with title=title_trans merged=1 %}
-            <ul class="tab-nav merged" data-tab-nav>
-                <li class="active"><a href="#">Tab1</a></li>
-                <li><a href="#">Tab2</a></li>
-            </ul>
+            <div class="w-tabs merged" data-tabs data-tabs-animate>
+                <div role="tablist" class="w-tabs__list">
+                    {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-1' title='Tab 1' active=True %}
+                    {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-2' title='Tab 2' %}
+                </div>
+            </div>
 
             <p>Tabs can also indicate errors:</p>
 
-            {% include "wagtailadmin/shared/header.html" with title=title_trans merged=1 %}
-            <ul class="tab-nav merged" data-tab-nav>
-                <li class="active"><a href="#" class="errors" data-count="123">Tab1</a></li>
-                <li><a href="#" class="errors" data-count="1">Tab2</a></li>
-            </ul>
+            <div class="w-tabs merged" data-tabs data-tabs-animate>
+                <div role="tablist" class="w-tabs__list">
+                    {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-1' title='Tab 1' active=True errors_count=5 %}
+                    {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-2' title='Tab 2' errors_count=5%}
```

Comment:
This isn’t working. Additionally, we shouldn’t set the same tab_id as above – they should be unique.

Reply:
Fixed!